src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -43,6 +43,20 @@
 
 logger = getLogger(__name__)
 
+# We allow some common aliases to be used
+OPTION_ALIASES = {
+    "HEADER": ("SKIP_HEADER", lambda val: 1 if val else 0),
+    "DELIMITER": ("FIELD_DELIMITER", lambda val: val),
+    "SEP": ("FIELD_DELIMITER", lambda val: val),
+    "LINESEP": ("RECORD_DELIMITER", lambda val: val),
+    "PATHGLOBFILTER": ("PATTERN", lambda val: val),
+    "QUOTE": ("FIELD_OPTIONALLY_ENCLOSED_BY", lambda val: val),
+    "NULLVALUE": ("NULL_IF", lambda val: val),
+    "DATEFORMAT": ("DATE_FORMAT", lambda val: val),
+    "TIMESTAMPFORMAT": ("TIMESTAMP_FORMAT", lambda val: val),
```

Comment:
Do you also want to add TIME_FORMAT?

Just to double confirm, these will be only used in the data load scenarios right? Asking since theses options won't take effect in other scenarios.

Reply:
these only apply in the context of DataFrameReader options not in any other scenarios. They are provided just for backwards compatibility and mapped to the closest equivalent in the current copy options